internal/engine/postgresql/pg_catalog.go

```diff
@@ -4150,16 +4150,6 @@ func genPGCatalog() *catalog.Schema {
 			},
 			ReturnType: &ast.TypeName{Name: "text"},
 		},
-		{
-			Name: "concat",
-			Args: []*catalog.Argument{
-				{
-					Type: &ast.TypeName{Name: "any"},
-					Mode: ast.FuncParamVariadic,
-				},
-			},
-			ReturnType: &ast.TypeName{Name: "text"},
-		},
```

Comment:
This concat is not in the pg_catalog on the main branch, so I removed it here too.